ExilenceNextApp/src/store/settingStore.ts

```diff
@@ -12,6 +12,8 @@ export class SettingStore {
   @persist
   @observable
   uiScale: number = electronService.webFrame.getZoomFactor() * 100;
+  @persist @observable logPath: string =
+    'C:/Program Files (x86)/Grinding Gear Games/Path of Exile/logs/Client.txt';
```

Comment:
Leaving this in since there are no way to accidently run events, we need to create both a background thread and set the path and then send the start event for the listener to start.